pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -725,12 +719,22 @@ func applyLoadBalancer(c *cluster.Cluster, lb *networking.LoadBalancerSettings,
 		c.LbPolicy = cluster.Cluster_CLUSTER_PROVIDED
 		c.ClusterDiscoveryType = &cluster.Cluster_Type{Type: cluster.Cluster_ORIGINAL_DST}
 	default:
-		c.LbPolicy = defaultLBAlgorithm()
+		ApplyDefaultLoadBalancer(c, lb)
```

Comment:
c.LbPolicy is unset?

Reply:
oh. Sorry. I thought I pushed that commit, but did not. Now pushed.